src/main/java/com/google/sps/server/CalendarUtils.java

```diff
@@ -54,8 +54,8 @@ private static LocalDate getCourseDate(LocalDate startDate, int timeInMin, int c
    */
   private static LocalTime getCourseTime(int timeInMin, int courseDayOfWeek) {
     int dayInHours = courseDayOfWeek * 24;
-    int hour = (int) Math.floor((timeInMin / 60) - dayInHours);
-    int minute = (int) (60 * (((timeInMin / 60.0) - dayInHours) - hour));
+    int hour = (int) ((timeInMin / 60) % 24);
+    int minute = (int) (timeInMin % 60);
```

Comment:
I don't think you need the type casting here, these expressions already evaluate to `int` types.

Reply:
Gotcha that makes sense, whoops, thank you!